docs/changelog.md

```diff
@@ -12,6 +12,12 @@ permalink: /changelog/
 * [Gecko](https://github.com/mozilla-mobile/android-components/blob/master/buildSrc/src/main/java/Gecko.kt)
 * [Configuration](https://github.com/mozilla-mobile/android-components/blob/master/buildSrc/src/main/java/Config.kt)
 
+* **feature-tab**
+  * ⚠️ **This is a breaking change**: Now `TabsUseCases.SelectTabUseCase` is an interface, if you want to use is previous behavior you could keep using  `TabsUseCases.selectTab` or us `TabsUseCases.DefaultSelectTabUseCase`
+
+* **feature-awesomebar**
+  * ⚠️ **This is a breaking change**: `SessionSuggestionProvider` now have a new parameter `excludeSelectedSession`, to ignore the selected session on the suggestions.
```

Comment:
nit: This shouldn't be a breaking change as `excludeSelectedSession` defaults to false, which is the same as before?

Reply:
Thanks for the catch, updated!